src/ifElse.test.js

```diff
@@ -1,11 +1,46 @@
 'use strict';
 
 describe('ifElse', () => {
-  // const { ifElse } = require('./ifElse');
+  const { ifElse } = require('./ifElse');
+  let firstCb;
+  let secondCb;
+  let condition;
 
-  it('should ', () => {
+  beforeEach(() => {
+    firstCb = jest.fn();
+    secondCb = jest.fn();
+    condition = jest.fn();
+  });
+
+  it('should be declared', () => {
+    expect(ifElse).toBeInstanceOf(Function);
+  });
 
+  it('should return undefined', () => {
+    expect(ifElse(condition, firstCb, secondCb)).toBeUndefined();
   });
 
-  // write tests here
+  it('should launch condition cb in any case', () => {
```

Comment:
Replace Cb with Callback in tests description as well )
fix it everywhere

Reply:
fixed